inputfiles/addedTypes.json

```diff
@@ -400,5 +400,46 @@
         "interface": "Element",
         "name": "getElementsByTagName",
         "signatures": ["getElementsByTagName(tagname: \"picture\"): NodeListOf<HTMLPictureElement>"]
+    },
+    {
+        "kind": "interface",
+        "name": "parentNode",
+        "flavor": "DOM",
+        "properties": [
+            {
+                "name": "children",
+                "type": "HTMLCollection"
```

Comment:
i would add a "read-only" property here and make it "1". thought we do not support this in the script now, we should soon. this way we get the correct declaration later on. that also applies to the other properties.